### PR TITLE
Added playbook to enable noop for the kernel IO queue scheduler.

### PR DESCRIPTION
**Pull request by curt-matthews**

### CODE REVIEW

File: playbooks/optimize_kernel_scheduler_for_vmware_guests.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+
+- name: Cloud | VMware | Enable noop kernel param for RHEL VMs on VMware
```
Comment (itewk) [inline playbooks/optimize_kernel_scheduler_for_vmware_guests.yml]: @curt-matthews out of matter of consistency i try to make the play name match the file name. So maybe change this to `Cloud | Optimize Kernal Scheduler` and then also rename the file to `optimize_kernel_scheduler.yml` that way later if we find a kernel scheduler optimization for AWS, or Azure, or Google, or RHV, or etc, we can just add them to this one playbook.

**Review by itewk — CHANGES_REQUESTED** [review]

**Comment by curt-matthews** [comment]
I've made the requested changes.

File: README.md [hunk 2]
```diff
@@ -40,3 +40,10 @@ The concept is that this playbook is cloud provider agnostic and can figure out
 | vsphere_username              | If vSphere   |                                |                                    | vSphere username for API calls
 | vsphere_password              | If vSphere   |                                |                                               | vSphere password for API calls
 | vsphere_datacenter            | If vSphere   |                                |                                               | vSphere datacenter for API calls. NOTE: attempted to determine this dynamically but could not find a way.
+
+### optimize_kernel_scheduler_for_vmware_guests.yml
```
Comment (itewk) [inline README.md]: update this to match the new playbook name

**Review by itewk — CHANGES_REQUESTED** [review]

**Review by itewk — APPROVED** [review]
looks good.

**Pull request merged**